### DISCUSSION TITLE
The GitHub Education application, approved on October 19 (IST), has not yet granted access to the GitHub Student benefits. Time of posting -> Oct 22 IST.

### DISCUSSION THREAD

**Discussion by AnshJain-Shwalia** [community/community#142192] [GitHub Education]
### Hi everyone,

I got my GitHub Education application approved on October 19 (IST), but I still don’t have access to the GitHub Student benefits. It's been a few days now (today’s October 22, IST), and I was hoping you could help me figure out what’s going on.

Thanks a lot for your help!

Best,
**Ansh Jain**
**jainansh1900@gmail.com**

**Comment by davevad93** [comment]
Hi @AnshJain-Shwalia ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AnshJain-Shwalia was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
